packages/eslint-plugin/docs/rules/no-useless-template-literals.mdx

```diff
@@ -9,53 +9,15 @@ import TabItem from '@theme/TabItem';
 >
 > See **https://typescript-eslint.io/rules/no-useless-template-literals** for documentation.
 
-This rule reports template literals that can be simplified to a normal string literal.
+This rule reports template literals that contain substitution expressions (also variously referred to as embedded expressions or string interpolations) that are unnecessary and can be simplified.
 
-## Examples
+:::warning
+This rule is being renamed to [`no-useless-template-expression`](./no-useless-template-expression.mdx).
```

Comment:
[Docs] I think this admonition orders things a little differently from how users would expect to read it. Maybe instead the order could be:

1. This rule is being renamed (as-is)
2. The current name will be removed
3. (blank line)
4. Deeper explanation

That way we get the pertinent info first, then the explanation later.

Reply:
I wrote this to parallel [the deprecation notice for no-useless-template-literal](https://typescript-eslint.io/rules/no-throw-literal/)

https://github.com/typescript-eslint/typescript-eslint/blob/0bd126fb44810b731eb8f336019c40a07964a0e1/packages/eslint-plugin/docs/rules/no-useless-template-literals.mdx#L14-L21


Should we update that too?

Reply:
Yeah! Good call :smil